src/Command/SearchImportCommand.php

```diff
@@ -65,8 +90,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $entitiesToIndex = array_unique($entitiesToIndex);
 
         foreach ($entitiesToIndex as $entityClassName) {
-            $manager    = $this->managerRegistry->getManagerForClass($entityClassName);
-            $repository = $manager->getRepository($entityClassName);
+            $manager       = $this->managerRegistry->getManagerForClass($entityClassName);
+            $repository    = $manager->getRepository($entityClassName);
+            $realIndexName = $this->searchService->searchableAs($entityClassName);
+
+            if ($shouldDoAtomicReidex) {
+                $tmpIndex = $this->searchServiceForAtomicReindex->searchableAs($entityClassName);
```

Comment:
Scout-extended is doing here try {} catch {}, but command does handle situation with missing original index fine according my testing.